package/boot/uboot-envtools/files/ath79

```diff
@@ -83,7 +83,8 @@ buffalo,wzr-hp-ag300h)
 	ubootenv_add_uci_config "/dev/mtd3" "0x0" "0x10000" "0x10000"
 	;;
 buffalo,wzr-hp-g300nh-rb|\
-buffalo,wzr-hp-g300nh-s)
+buffalo,wzr-hp-g300nh-s|\
+linksys,ea4500-v3)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x20000" "0x20000"
```

Comment:
Just some information, if this part is wrong, when you execute `fw_printenv` you'll get a wrong crc warning.

Reply:
Done.